server/routes/api/dashboard.js

```diff
@@ -31,6 +31,40 @@ router.get('/getCoursesByStudent', (req, res) => {
     });
 });
 
+
+/**drops a section by a given student, such that it removes him from the registered_students list
+ * of a given course
+ * ==========================================
+ * Example api call body:
+ * {
+    "studentID": "5f984a44da9eb32ba01d31dd",
+    "sectionID" : "5f984a44da9eb32ba01d31df"
+    }
+ */
+router.put('/dropSection', (req, res) => {
```

Comment:
Also the `dropSection` endpoint should be in the 'section'  API, not 'dashboard' since students will drop the course from within the course view, not the dashboard. 
It should also be a POST (a PUT is used when an endpoint can be called more than once with the same input without anything happening after the first call. In this case calling it more than once causes an error since the student won't be in the course anymore). source: https://restfulapi.net/rest-put-vs-post/